test/DurableTask.Netherite.Tests/DurableTask.Netherite.Tests.csproj

```diff
@@ -10,6 +10,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.11.1" />
+	<PackageReference Include="Moq" Version="4.18.4" />
```

Comment:
nit: another indentation that is off. Try to use only spaces (no tabs) since tabs render differently in different environments.